src/display/api.js

```diff
@@ -1128,7 +1129,7 @@ class PDFDocumentProxy {
  * @property {Array<TextItem | TextMarkedContent>} items - Array of
  *   {@link TextItem} and {@link TextMarkedContent} objects. TextMarkedContent
  *   items are included when includeMarkedContent is true.
- * @property {Object<string, TextStyle>} styles - {@link TextStyle} objects,
+ * @property {Record<string, TextStyle>} styles - {@link TextStyle} objects,
```

Comment:
Please note that the primary use-case for the JSDoc-comments is to generate https://mozilla.github.io/pdf.js/api/draft/index.html, and consequently we can't have any TypeScript-specific types here; please just revert this change.